Diggy/PressureMap.lua

```diff
@@ -1,12 +1,26 @@
 -- dependencies
+local Event = require 'utils.event'
 
 -- this
 local PressureMap = {}
 
+-- main block
 global.pressure_map_storage = {}
+local defaultValue = 1
+local _mt_y = { __index=function(tbl,key) tbl[key] = defaultValue return tbl[key] end}
+local _mt_x = {__index=function(tbl,key) tbl[key] = setmetatable({},_mt_y) return rawget(tbl,key) end}
+
+Event.on_load(function()

```

Comment:
We need on_load because the meta tables arent serialized. So if a user downloads the map and loads it they have to set the metatable, otherwise it will desync.

If the metatable is set 3 times it wont make a difference. But i'm 80% sure code inside a file is only executed once. Maybe i'm wrong, we would have to try. 

I just commited something else tho: on_load isnt called on init, so i added that